### PR TITLE
DR-2599 deploy to migrated production environment

### PR DESCRIPTION
**Pull request by chrisb87**
This allows merges to the production branch to dual-deploy to the old and new clusters.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ services:
 - postgresql
 - mysql
 - docker
+branches:
+  only:
+  - qa
```
Comment (croyfish) [inline .travis.yml:9]: 👍

File: .travis.yml [hunk 2]
```diff
@@ -26,4 +30,7 @@ env:
...
+  - secure: "oKbK1zNQMUPOWMsddb/TB2A8PHjrc8s00R6z9oJqtsEnX1YVHEzsZjIRsWhCaNz6wuumrg9Dl1n5YYu71libKDhq2AeOx1f1J203nns48oaaGJnnXRqcT1I9w/PN7FB7cHldRmijvYTXgYVNkYz9MAg8VSLsw3nslGyzvK93DozZNcF+G70RWDr6FazHta3FIFbhZNsktmPlCcCITe4ik0M97Xbchwr3chnpcMmRrr+Ph47yqGsOeMHUxf4wjlF34aZpNjTsxKI2YRTAw9X+S1X6/FG+9eKe0DGUvap8OjVYRpO7bC5B/P1/y/yGM1P+bIB0hZ12Qtx18iAk2SWdzK899L4gtw6WuuovByjbOohAdAoyr32w2Qj9n1ms0ZWaa66nQ+dJEEuMmFTpQ9AlXbfkI7by+G8YYnu5IC9HX1RtYQOUkXkwpoEG7eLXNqrwsPh0/SaijodhjO90faCb5Gag9FMOPFWw0VktE/s/5/88JqvUH4JLzDBvxoLyqXAAL0CVkhL2R3xwYikdQjBfii7ZyXytQ0sm7lpyqvIG7elJSOc7PRSWqWJJ7GTgUSAQdfldv2On2Mbux1ET2p6QC8I6KXDNSM/bTvRFI2Hl9OZC5z1F91gTg386QIBcNU6263Zh7+IHE9nR/MYMkPnKLvqLZ1Rn+SA2T9Ez+QT4/DU=" # AWS_SECRET_ACCESS_KEY_PRODUCTION_NEW 11/02/2023
   - REMOTE_IMAGE_URL_QA_NEW=685731035297.dkr.ecr.us-east-1.amazonaws.com/fedora-ingest
+  - REMOTE_IMAGE_URL_PRODUCTION_NEW=557492996044.dkr.ecr.us-east-1.amazonaws.com/fedora-ingest
```
Comment (croyfish) [inline .travis.yml:37]: Should be consistent between accounts, but we'll have to ensure this is actually what this repo gets named

File: provisioning/travis_ci_and_cd/ecs_deploy_new.sh [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
-    export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY_QA_NEW
-    CLUSTER_NAME='fedora-ingest-rails-qa'
-    WEB_APP_SERVICE_NAME='fedora-ingest-rails-web-application-qa'
-    WORKER_SERVICE_NAME='fedora-ingest-rails-worker-qa'
+  if [ "$TRAVIS_BRANCH" == "qa" ] || [ "$TRAVIS_BRANCH" == "production" ]; then
+    case "$TRAVIS_BRANCH" in
+      production)
+        export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID_PRODUCTION_NEW
+        export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY_PRODUCTION_NEW
+        CLUSTER_NAME='fedora-ingest-rails-production'
```
Comment (croyfish) [inline provisioning/travis_ci_and_cd/ecs_deploy_new.sh:9]: 👍 Confirmed that this is the named being used

**Review by croyfish — COMMENTED** [review]
Didn't we want to cut a new production branch nypl-dams-prod like we've been doing for the other applications? (https://jira.nypl.org/browse/DR-2599)

**Comment by chrisb87** [comment]
> Didn't we want to cut a new production branch nypl-dams-prod like we've been doing for the other applications? (https://jira.nypl.org/browse/DR-2599)

You're right. I did this slightly differently, having pushes to the production branch deploy to both. That's not what we want! Will fix that up.

File: .travis.yml (hunk 2 above)
Comment (chrisb87) [reply .travis.yml:37]: The repo has been created, this is the right name :+1:

**Review by croyfish — APPROVED** [review]
👍 Looks good and thinking we could merge before it's testable since we need the code to be on the nypl-dams-prod branch anyway.

**Pull request merged**